### PR TITLE
pass campaign url params

### PR DESCRIPTION
**Pull request by felicio**
## why

- https://github.com/status-im/keycard-site/issues/83

### CODE REVIEW

File: themes/keycard/layout/index.ejs [hunk 1]
```diff
@@ -199,3 +199,11 @@
...
   </div>
 </main>
+
+<script>
+  /* pass campaign url params to Shopify on Get Keycard CTA https://github.com/status-im/keycard-site/issues/83 */
+  document.querySelectorAll('a[href="https://get.keycard.tech/"]').forEach(a => {
+    let p = location.search.match(/utm_[^=&]+=[^&]+/g);
+    if (p) a.href += (a.href.includes('?') ? '&' : '?') + p.join('&');
+  });
+</script>
```
[the remark this answers is not in the document]
Comment (marcelines) [reply themes/keycard/layout/index.ejs:209]: Non-blocking nevertheless.

**Review by marcelines — APPROVED** [review]

File: themes/keycard/source/javascripts/utm-handler.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Pass campaign url params to Shopify on Get Keycard CTA https://github.com/status-im/keycard-site/issues/83
+
+function storeUtmParams() {
+  const utmParams = new URLSearchParams(window.location.search)
+    .toString()
+    .match(/utm_[^=&]+=[^&]+/g)
+
+  console.log('utmParams', utmParams)
```
Comment (marcelines) [inline themes/keycard/source/javascripts/utm-handler.js:8]: Delete console.log?

**Review by jkbktl — APPROVED** [review]

**Review by guylouis — APPROVED** [review]

**Comment by jakubgs** [comment]
Did you verify via https://dev.keycard.tech/ site that the version in `develop` works as expected?

**Comment by jakubgs** [comment]
Guys, have you heard about squashing commits?

image

This is unsightly.

**Comment by felicio** [comment]
> Did you verify via https://dev.keycard.tech/ site that the version in `develop` works as expected?

No. As you pointed out in DMs, `dev` wasn't being deployed, only prod was. Which worked as expected.

**Comment by felicio** [comment]
> squashing commits?

Not a maintainer of this repo, just helped out. Can you set rule accordingly in repo's settings?

**Comment by jakubgs** [comment]
I think rebase merging is fine a long as you keep your PR commits tidy.

**Pull request merged**